packages/react-table/src/components/TableComposable/TableComposable.tsx

```diff
@@ -71,7 +71,6 @@ const TableComposableBase: React.FunctionComponent<TableComposableProps> = ({
   ouiaSafe = true,
   isTreeTable = false,
   isNested = false,
-  nestedHeaderColumnSpans,
```

Comment:
I believe you still need this so it does not get spread to the table and cause errors.